spec/policies/user_policy_spec.rb

```diff
@@ -33,6 +33,9 @@
     it { is_expected.not_to permit_action(:update) }
     it { is_expected.not_to permit_action(:edit) }
     it { is_expected.not_to permit_action(:destroy) }
+    it { expect(subject.send(:admin?)).to be false }
```

Comment:
You could move them into the public scope :smile: 

I'm just reviewing Code Climate and the Pundit spec helpers to understand why those methods are not getting coverage. It is suggesting to me that the Pundit spec helpers are not following the expected call paths.